appwrite/website
# Add sign-in with github blog post

## What does this PR do?

- Adds Sign in with GitHub blog post
- Add blog post assets

## Test Plan

- Run the website and navigate to `/blog/post/implement-sign-in-with-github
- Confirm that blog post exists and that the assets are displayed correctly

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes
